templates/http/httpfactory.h.tpl

```diff
@@ -3,7 +3,7 @@
 #include <QtCore>
 #include <QtNetwork>
 
-#include "{{snake .Module.Name}}/api/apifactory.h"
+#include "{{snake .Module.Name}}/api/apifactoryinterface.h"
```

Comment:
wasn't another class named "I<classname>.h" why this "...interface.h"

Reply:
you mean I should rename it to iapifactory and a class to IApiFactory?